lib/pliny/middleware/versioning.rb

```diff
@@ -44,13 +44,14 @@ def detect_api_version(env)
       env['HTTP_ACCEPT'] = media_types.join(', ')
 
       version ||= @default
-      set_api_version(env, version)
+      set_api_version(env, version, variant)
       @app.call(env)
     end
 
-    def set_api_version(env, version)
+    def set_api_version(env, version, variant)
       # API modules will look for the version in env
       env["HTTP_X_API_VERSION"] = version
```

Comment:
I know changing the `X-` thing above might cause a breakage for this particular header. I wonder how we should migrate over, 1) just do it, 2) support both, 3) only stick to `X-`, 4) ???, 5) profit.